src/client/jest.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   moduleNameMapper: {
     "^@/(.*)$": "<rootDir>/src/$1",
   },
-  testPathIgnorePatterns: ["<rootDir>/e2e/"],
+  testPathIgnorePatterns: ["<rootDir>/e2e/", "utils.ts"],
```

Comment:
losing `jest` soon, so this may be irrelevant, but shouldn't jest just be picking up spec/test files .. and stuff in __tests__, so I would argue `utils.ts` does not belong in `__tests__` as it isn't a test ?
.. bit of a minor point, I'd agree .. suggest a `testUtils` folder, if we still need it post-migration to `vitest`

Reply:
Yeah, this felt weird as I was doing it, I'll move it to a different folder